### ISSUE TITLE
Fetch bus connectors

### ISSUE THREAD

**Issue by adeas31** [OpenModelica/OMSimulator#382]
**Is your feature request related to a problem? Please describe.**
Support for bus connectors are added recently via `oms3_addBus`. However, it is not possible to load the saved bus connectors via `oms3_import`.

**Describe the solution you'd like**
It should be possible to load the saved/created bus connectors.

**Describe alternatives you've considered**
One idea is to extend the `oms3_element_t` like this,
```
typedef struct _oms3_element_t{
  oms3_element_enu_t type;           ///< Element type, i.e. system or component
  char* name;                        ///< Name of the element
  struct _oms3_element_t** elements; ///< List (null-terminated array) of all sub-elements
  oms_connector_t** connectors;      ///< List (null-terminated array) of all interface variables: inputs, outputs, and parameters.
  oms3_busconnector_t** busConnectors;      ///< List (null-terminated array) of all bus connectors
  ssd_element_geometry_t* geometry;  ///< Geometry information of the element
} oms3_element_t;

```
Another idea is to add a new API `oms3_getBusConnectors` or `oms3_getBuses`.

**Comment by robbr48** [comment]
Are you sure it is not working? This should already be implemented in #379 and tested by the "import_export.lua" test model.

**Comment by adeas31** [comment]
What I mean is there is no way to get this information via C-API.

**Comment by adeas31** [comment]
Fixed in #384. @robbr48 Thanks for the quick fix.
